frontend/public/components/monitoring/_monitoring.scss

```
@@ -130,6 +130,33 @@
   padding-right: 0;
 }
 
+.query-browser__tooltip {
+  background-color: #151515;
+  color: #eee;
+  overflow: scroll;
+  padding: 15px;
+  z-index: 999999;
```

maybe

```suggestion
  z-index: $zindex-tooltip;
```

although there's probably a PF4 var that's better

cc @rhamilto

---

https://github.com/openshift/console/pull/2101 resolves this by removing the `z-index`.